README.md

```diff
@@ -457,70 +462,118 @@ To debug issues you can consult logs of following components (commands are for l
    ```
 ### Logging in Grafana
 
-Logs for extension are available in Grafana in namespace garden. Logs for cri-resmgr-installation, cri-resmgr-agent and cri-resource-manager in seed grafana.
+Logs for extension are available in Grafana in namespace garden. Logs for cri-resmgr-installation, cri-resmgr-agent and cri-resource-manager in seed grafana (commands below assume running local setup)
 
-##### cri-extension
+#### cri-resmgr-extension
 You can check the logs in Grafana - svc/grafana namespace garden.
 
-Loki expression 
-> {container_name="gardener-extension-cri-resmgr"}
+Expose it first with:
+```
+kubectl port-forward -n garden svc/grafana 3000:3000
+```
+Then go to http://127.0.0.1:3000 and Dashboards/Extensions or (generic Pod Logs) and choose "gardener-extension-cri-resmgr"
 
-Dashboard grafana
+Dashboard grafana:
 
 <a href="./doc/cri-resmeg-extension.png">
 <img src="./doc/cri-resmeg-extension.png" width="80%" height="80%">
 </a>
 
-##### cri-rm-installation and cri-rm-agent
-You can check the logs in Grafana - svc/grafana-operators in seed namespace.
+or Explore with Loki expression 
+```
+{container_name="gardener-extension-cri-resmgr"}
+```
+
+to filter only errors with following query:
+```
+{container_name="gardener-extension-cri-resmgr"}|json|log_level="error"|line_format "{{.log_msg}} {{.log_error}}"
+```
+
+#### cri-resmgr-installation and cri-resmgr-agent
+
+You can check the logs in Grafana for operators "grafana-operators" in Shoot namespace.
+
+Expose it first with (for shoot local)
+```
+kubectl port-forward -n shoot--local--local svc/grafana-operators 3001:3000
+```
+Then go to http://127.0.0.1:3001 and Kubernetes DaemonSets and chose "cri-resmgr-agent" or "cri-resmgr-installation" then drill down to specific Pod
+
+
+| Bug in kind based local gardener Grafana operator Loki configuration|
+|---------------------------------------------------------------|
+| For local kind-based setup, HTTP header is missing: `X-Scope-OrgID: operator`. To fix it do the following, first sign in with `admin/admin` and then:
+<a href="./doc/fix_grafana_loki.png"> <img src="./doc/fix_grafana_loki.png" width="80%" height="80%"> </a> |
+
+Or use Explorer and entry following Loki queries for **installation** logs:
+
+```
+{pod_name=~"cri-resmgr-installation-.*", container_name="installation"}
+```
+
+and for **agent**:
+```
+{pod_name=~"cri-resmgr-agent-.*"}
+```
 
-Loki expression
->{pod_name="cri-resmgr-installation-XXXXX"} 
->{pod_name="cri-resmgr-agent-XXXXX"} 
+Example screenshots below:
```

Comment:
Make it a header or make it bold. Do something to indicate that the content below contains example screenshots you mention. Otherwise, it looks like something is missing

Reply:
added